docs: Update analytics events update and new analytic events templates (#4447)
chanzuckerberg/single-cell-data-portal: .github/ISSUE_TEMPLATE/analytics-events-update.md

---
name: Analytics events update
about: Used to submit requests for updates to current analytic events
title: 'Analytics (update): <fill in description> '
labels: analytics, frontend, tech
assignees: ''

---

### Overview
**Tool**: `<Which tool are these events for, i.e. gene expression, explorer>`
Feature: `<Which feature in that tool are these events for, i.e. filters>`

**Description**: `<Reason for update>`

### Updates needed
`<Event name>`
- Current: `<Description of how the event works currently>`
- Requested change: `<Description of the update needed>`

Checklist 

> - [ ] Once event updates are included in Staging ping @ainfeld to check analytics 

Reviewer: ```suggestion
- [ ] Once event updates are deployed to Staging ping @ainfeld to check analytics 
```

- [ ] Verify events are showing up in the Staging space in Plausible correctly 

> - [ ] Once event updates are included in Prod ping @ainfeld 

Reviewer: ```suggestion
- [ ] Once event updates are deployed to Prod ping @ainfeld 
```
